app/api/management_api/v1/otp.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'barong/security/access_token'
+
+module ManagementAPI
+  module V1
+    class OTP < Grape::API
+      helpers do
+        def generate_signed_payload(payload)
+          require 'openssl'
+          keychain = { barong: Barong::Security.private_key }
+          algorithms = { barong: 'RS256' }
+          JWT::Multisig.generate_jwt(payload, keychain, algorithms)
```

Comment:
Try to modify existing JWT using https://github.com/rubykube/jwt-multisig/blob/master/lib/jwt-multisig.rb#L65